http://dominion.thruhere.net/koen/cms/recommended-kernel-config-options-for-a-modern-angstrom-system
old: -rw-r--r-- 1 mk users 3189608  1. Apr 22:14 git/arch/arm/boot/uImage
new: -rw-r--r-- 1 mk users 3205580 28. Mai 11:43 git/arch/arm/boot/uImage
--- .config.orig	2012-05-28 11:14:29.000000000 +0200
+++ .config		2012-05-28 11:25:25.000000000 +0200
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.36.2
-# Sun Apr  1 17:06:26 2012
+# Mon May 28 11:25:25 2012
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -53,7 +53,10 @@
 CONFIG_POSIX_MQUEUE=y
 CONFIG_POSIX_MQUEUE_SYSCTL=y
 # CONFIG_BSD_PROCESS_ACCT is not set
-# CONFIG_TASKSTATS is not set
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_XACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
 # CONFIG_AUDIT is not set
 
 #
@@ -750,7 +753,8 @@
 # Generic Driver Options
 #
 CONFIG_UEVENT_HELPER_PATH=""
-# CONFIG_DEVTMPFS is not set
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_STANDALONE=y
 CONFIG_PREVENT_FIRMWARE_BUILD=y
 CONFIG_FW_LOADER=y
@@ -2137,9 +2141,9 @@
 CONFIG_INOTIFY_USER=y
 # CONFIG_QUOTA is not set
 # CONFIG_AUTOFS_FS is not set
-# CONFIG_AUTOFS4_FS is not set
+CONFIG_AUTOFS4_FS=y
 CONFIG_FUSE_FS=y
-# CONFIG_CUSE is not set
+CONFIG_CUSE=y
 
 #
 # Caches
